Source/Core/Common/StringUtil.cpp

```diff
@@ -219,7 +219,7 @@ std::string ArrayToString(const u8* data, u32 size, int line_len, bool spaces)
   return oss.str();
 }
 
-// Turns "  hello " into "hello". Also handles tabs.
+// Turns "\n\r\t hello " into "hello".
```

Comment:
Since you're changing the comment, maybe it should include "at the start/end but not inside". This appears to be what "trim" is in other languages, not "normalize space" (which is my immediate thought going by the name).